src/rabbit_auth_backend_internal.erl

```diff
@@ -165,7 +165,19 @@ permission_index(read)      -> #permission.read.
 %%----------------------------------------------------------------------------
 %% Manipulation of the user database
 
+validate_credentials(_Username, Password) ->
+    rabbit_credential_validation:validate_password(Password).
```

Comment:
If we pass both then we need to also rename the built-in validators to reflect that they only validate passwords. OK.